grafana-plugin/src/components/SortableList/SortableList.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 
-import { SortableContainer } from 'react-sortable-hoc';
+import { SortableContainer, SortableContainerProps } from 'react-sortable-hoc';
 
 import { Timeline } from 'components/Timeline/Timeline';
 
 export const SortableList = SortableContainer(({ className, children }: any) => {
   return <Timeline className={className}>{children}</Timeline>;
-});
+}) as React.ComponentClass<SortableContainerProps & { className: string; children: React.ReactNode[] }>;
```

Comment:
We can pass type as generic and avoid casting
```
export const SortableList = SortableContainer<{ className: string; children: React.ReactNode[] }>(
  ({ className, children }: any) => {
    return <Timeline className={className}>{children}</Timeline>;
  }
);
```

Reply:
Missed the `any` param, fixed it.